docs/source/basic-usage.rst

```diff
@@ -58,25 +58,27 @@ into higher level representations of WebSocket events. In ``<APPLICATION
 GLUE>``, you need to write code to process these events. The
 :class:`WSConnection <wsproto.connection.WSConnection>` class contains a
 generator method :meth:`events <wsproto.connection.WSConnection.events>` that
-yields WebSocket events. To send a message, you call the :meth:`send_data
-<wsproto.connection.WSConnection.send_data>` method.
+yields WebSocket events. To send a message, you call the :meth:`send
+<wsproto.connection.WSConnection.send>` method.
 
 Connecting to a WebSocket server
 --------------------------------
 
-Begin by instantiating a connection object. The ``host`` and ``resource``
-arguments are required to instantiate a client. If the WebSocket server is
-located at ``http://myhost.com/foo``, then you would instantiate the connection
-as follows::
+Begin by instantiating a connection object in the client mode and then
+create a :class:`Request <wsproto.events.Request>` class to send. The
+Request must specify ``host`` and ``resource`` arguments. If the
+WebSocket server is located at ``http://myhost.com/foo``, then you
+would instantiate the connection as follows::
 
-    ws = WSConnection(ConnectionType.CLIENT, host="myhost.com", resource='foo')
+    ws = WSConnection(ConnectionType.CLIENT)
+    ws.send(Request(host="myhost.com", target='foo'))
 
 Now you need to provide the network glue. For the sake of example, we will use
 standard Python sockets here, but `wsproto` can be integrated with any network
 layer::
 
     stream = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    stream.connect(("myhost", 8000))
+    stream.connect(("myhost", 80))
```

Comment:
Lets use the same host as above: `myhost.com`.
Thinking about it, why not use the generic/common `example.com` to prevent problems?